.cirrus.yml

```diff
@@ -65,3 +66,79 @@ vagrant_task:
     else
       ssh -tt default "sudo -i make -C /vagrant localrootlessintegration"
     fi
+
+task:
+  timeout_in: 30m
+  env:
+    HOME: /root
+    CIRRUS_WORKING_DIR: /home/runc
+    GO_VERSION: "1.16.4"
+    BATS_VERSION: "v1.3.0"
+    # yamllint disable rule:key-duplicates
+    matrix:
+      DISTRO: centos-7
+      DISTRO: centos-stream-8
+
+  name: ci / $DISTRO
+  compute_engine_instance:
+    image_project: centos-cloud
+    image: family/$DISTRO
+    platform: linux
+    cpu: 4
+    memory: 8
+
+  install_dependencies_script: |
+    if [ $DISTRO == centos-7 ]; then
+      yum install -y -q epel-release
+      (cd /etc/yum.repos.d && curl -O https://copr.fedorainfracloud.org/coprs/adrian/criu-el7/repo/epel-7/adrian-criu-el7-epel-7.repo)
+    elif [ $DISTRO == centos-stream-8 ]; then
+      yum install -y -q https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm dnf-plugins-core
```

Comment:
It seems that epel-release is available in centos-stream-8 repo (in extras): http://mirror.centos.org/centos/8-stream/extras/x86_64/os/Packages/

So maybe
```bash
yum --enablerepo extras install -y -q epel-release dnf-plugins-core
```